src/elements/common/sub-header/Add.js

```diff
@@ -24,12 +24,12 @@ const Add = ({ onUpload, onCreate, showUpload = true, showCreate = true }: Props
         <AddButton />
         <Menu>
             {showUpload && (
-                <MenuItem onClick={onUpload}>
+                <MenuItem dataTestId="be-btn-add-upload" onClick={onUpload}>
```

Comment:
We should try to select these and other elements via their user-facing text content, `messages.upload`, rather than data-testid whenever possible.